xDrip/Resources/en.lproj/Localizable.strings

```diff
@@ -454,7 +454,7 @@ This setting will be applied to all future sensors";
 // Bluetooth Errors
 "bluetooth_error_title" = "Transmitter Service Error";
 "bluetooth_error_unknown" = "Unknown bluetooth error happened";
-"bluetooth_error_powered_off" = "Bluetooth is powerered off";
+"bluetooth_error_powered_off" = "Enable Bluetooth and try again";
```

Comment:
Let's say "Bluetooth is powered off. Please enable it in system settings and try again".